language/diem-framework/modules/CRSN.move

```diff
@@ -0,0 +1,310 @@
+/// A module implementing conflict-resistant sequence numbers (CRSNs).
+/// The specification, and formal description of the acceptance and rejection
+/// criteria, force expiration and window shifting of CRSNs is described in DIP-168.
+module 0x1::CRSN {
+    use 0x1::BitVector::{Self, BitVector};
```

Comment:
I think we can still call `BitVector` methods without the `Self` important (but could be wrong)

Reply:
Still need the Self import in order to call `BitVector` functions -- this is similar to `self` imports in Rust.